pkg/oc/cli/cmd/debug.go

```diff
@@ -362,12 +362,23 @@ func (o *DebugOptions) Debug() error {
 			if stderr == nil {
 				stderr = os.Stderr
 			}
+
+			podResult, err := o.Attach.PodClient.Pods(pod.Namespace).Get(pod.Name, metav1.GetOptions{})
+			var rc int
+			if err != nil {
+				fmt.Fprintf(stderr, "error: unable to get the pod return code.")
+				rc = 1
+			} else {
+				rc = int(podResult.Status.ContainerStatuses[0].State.Terminated.ExitCode)
```

Comment:
I think it might be worth checking for `len(podResult.Status.ContainerStatuses)`

Reply:
Also, `podResult.Status.ContainerStatuses[*].State.Terminated` can be nil, causing a panic, and the debug pod not to be deleted. Worth checking for that as well. Maybe do:
```go
if err == nil && len(podResult.Status.ContainerStatuses) > 0 && podResult.Status.ContainerStatuses[0].State.Terminated != nil {
    rc = int(podResult.Status.ContainerStatuses[0].State.Terminated.ExitCode)
} else {
    fmt.Fprintf(stderr, "error: unable to get the pod return code.")
    rc = 1
}
```

Reply:
If no -c flag isn't provided only the first one will be used so **IF I understand it correclty** it's safe  be safe.

However, if the -c flag is provided then it'd be necessary to iterate over the containerStatutes and this should be blocked until implemented.

Reply:
Also, is it possible under any scenario that we get a 0 length causing the program to read out of index?